clevertap-core/src/main/java/com/clevertap/android/sdk/utils/PackageUtils.java

```diff
@@ -53,38 +51,4 @@ private static boolean isIntentResolved(Context context, Intent intent) {
         return (intent != null
                 && context.getPackageManager().resolveActivity(intent, PackageManager.MATCH_DEFAULT_ONLY) != null);
     }
-
-    /**
-     * Check if device is xiaomi, running MIUI OS
-     * @param context application context
-     * @return true if device is xiaomi, running MIUI OS or false
-     */
-    public static boolean isXiaomiDeviceRunningMiui(Context context) {
-        try {
-            String manufacturer = "xiaomi";
-            if (!manufacturer.equalsIgnoreCase(android.os.Build.MANUFACTURER)) {
-                return false;
-            }
-
-            @SuppressLint("PrivateApi")
-            Class<?> c = Class.forName("android.os.SystemProperties");
-            Method get = c.getMethod("get", String.class);
-            String miui = (String) get.invoke(c, "ro.miui.ui.version.code");
-            if (miui!=null && !TextUtils.isEmpty(miui.trim()))
-            {
-                return true;
-            }
-        } catch (Throwable t) {
-            t.printStackTrace();
-        }
-
-        return isIntentResolved(context,
```

Comment:
@CTLalit we can remove `isIntentResolved ()` also as it's used by `isXiaomiDeviceRunningMiui` only.

Reply:
done